<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hospital Management System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="Table">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>911</width>
      <height>371</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QComboBox" name="PanelSelector">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>911</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="AddRecordButton">
       <property name="text">
        <string>Add New Record</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteSelectedRow">
       <property name="text">
        <string>Delete Selected ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OrderByDate">
       <property name="text">
        <string>Order By Date</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>540</y>
      <width>911</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="tableCount">
       <property name="text">
        <string>Element Count: 000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="earliestapp">
       <property name="text">
        <string>Earliest Appointment: time id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="latestapp">
       <property name="text">
        <string>Latest Appointment: time id</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="mostappdoc">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>580</y>
      <width>298</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>Most Appointed Doctor: 0 0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
